.github/workflows/release-to-pypi.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4.3.0
         with:
-          python-version: 3.9
+          python-version: 3.10
```

Comment:
Pyodide uses [Python 3.11](https://blog.pyodide.org/posts/0.23-release/) (and [release candidates 0.26](https://github.com/pyodide/pyodide/issues/4420) have been built against 3.12) so perhaps worth building with those to ensure everything works.